data-processing/data_manipulation/config.yml

```diff
@@ -1,22 +1,21 @@
 minio:
-  access_key: '${MINIO_ACCESSKEY: hpU4SCmj5jiU7IP5}'
-  secret_key: '${MINIO_SECRETKEY: 7AUewBESqvKijdnNskm8nU6emTZ3rG8F}'
-  api_url: '${MINIO_API_URL: kubeagi-minio.172.22.96.136.nip.io}'
+  access_key: '${MINIO_ACCESSKEY: xxxxxx}'
+  secret_key: '${MINIO_SECRETKEY: xxxxxx}'
+  api_url: '${MINIO_API_URL: xxxxxx}'
   secure: '${MINIO_SECURE: True}'
   dataset_prefix: '${MINIO_DATASET_PREFIX: dataset}'
 
 zhipuai:
-  api_key: '${ZHIPUAI_API_KEY: 871772ac03fcb9db9d4ce7b1e6eea210.VZZVy0mCox0WrzQI}'
+  api_key: '${ZHIPUAI_API_KEY: xxxxxx}'
 
 llm:
   use_type: '${LLM_USE_TYPE: open_ai}' # zhi_pu_online or open_ai
-  qa_retry_count: '${LLM_QA_RETRY_COUNT: 100}'
+  qa_retry_count: '${LLM_QA_RETRY_COUNT: 10}'
 
 open_ai:
   key: '${OPEN_AI_DEFAULT_KEY: fake}'
-  base_url: '${OPEN_AI_DEFAULT_BASE_URL: http://arcadia-fastchat.172.22.96.167.nip.io/v1}'
-  model: '${OPEN_AI_DEFAULT_MODEL_NAME: 3d407a8b-90ab-43c4-9fc0-87e533368570}'
-
+  base_url: '${OPEN_AI_DEFAULT_BASE_URL: xxxxxx}'
+  model: '${OPEN_AI_DEFAULT_MODEL_NAME: xxxxxx}'
```

Comment:
if we allow user to select model service, do we still need configuration like zhipuai，llm.use_type and open_ai?

Reply:
don't need